rootfs/api/models.py

```diff
@@ -152,7 +142,7 @@ class App(UuidAuditedModel):
     """
 
     owner = models.ForeignKey(settings.AUTH_USER_MODEL)
-    id = models.SlugField(max_length=24, unique=True, default=select_app_name,
```

Comment:
Yeah - Django 1.7 changes the application loading logic and does so not like it that we were accessing `App` in `select_app_name` and then throwing it into the `SlugField` - It has to be with loading of imports / operations and how they want to guarantee consistency and availability across the board

This caused the test runner (and others) to crap out and say "AppRegistry is not ready"

https://docs.djangoproject.com/en/1.7/releases/1.7/#app-loading-refactor is the relevant bit of the changelog
> Django imports all application configurations and models as soon as it starts, through a deterministic and straightforward process. This should make it easier to diagnose import issues such as import loops.